### ISSUE TITLE
[Feature Request] Improve customization of commands with table output

### ISSUE THREAD

**Issue by InspectorCaracal** [evennia/evennia#3552]
#### Is your feature request related to a problem? Please describe.
There are a large number of default/built-in commands that use styled tables or dividers to display their output: `who`, `scripts`, `examine`, etc.

There is also a method specifically attached to the base command class, `styled_table`, that theoretically lets you generate tables influenced by user preferences and which could be easily customized on your base Command class to apply to your whole game.

Similarly, there are `styled_header`, `styled_separator`, and `styled_footer`, which are intended to be used the same way.

The problem is, the actual *use* of these methods is inconsistent in the built-in commands, leading to a lot of missed cases when trying to override those methods to modify the output of base commands without fully reimplementing them. For example, `CmdWho` uses `styled_table`, but `CmdScripts` does not. And `CmdExamine` has its own implementation for rendering separators with its own, completely independent class attributes instead of using the pre-existing `styled_separator` method.

#### Describe the solution you'd like
The default commands should all reference these methods rather than having their own custom implementations of table styling and other decorators, to be more consistent and more easily customized across an entire game.

**Comment by Griatch** [comment]
Yeah, the styles were added much later than most of these commands, so one would just need to sit down and homogenize them all.
